docs/configuration/http_conn_man/headers.rst

```diff
@@ -108,6 +108,39 @@ This header is populated by the :ref:`Ip Tagging Filter<config_http_filters_ip_t
 
 .. _config_http_conn_man_headers_x-forwarded-for:
 
+x-forwarded-client-cert
+---------------
+
+*x-forwarded-clinet-cert* (XFCC) is a proxy header which indicates certificate information of part
+or all of the clients that a request has flowed through on its way from the client to the server.
+A proxy may choose to sanitize/append/forward the XFCC header before proxying the reqeust.
+
+The XFCC header value is a comma (",") separated string. Each substring is an XFCC element, which
```

Comment:
Can any of the key/values contains `=` or `,`? Is there a need to consider escaping?